app/controllers/staff_area_controller.rb

```diff
@@ -10,4 +11,12 @@ def ensure_staff
     end
   end
 
+  def default_space
+    begin
+      @space = current_user.lab_sessions.last.space
+    rescue NoMethodError => e
```

Comment:
I'm really not a fan of rescue like this. You can use the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) to do something like this `@space = current_user.lab_sessions&.last&.space || Space.first`

Reply:
done